### DISCUSSION TITLE
Password add totp

### DISCUSSION THREAD

**Discussion by ljlzzzzz** [dani-garcia/vaultwarden#4466] [Q&A] [answered]
Hello, author, does the password support adding totp to the login password? Because my current login password is the original password add totp together, not separate ones.

**Comment by kpfleming** [comment, marked as answer]
No, the Bitwarden clients do not have a method of providing the stored password with the TOTP token appended to it. You'll have to do that yourself by using something like Ctrl-Shift-L to fill in the username and password fields, and then Ctrl-V (after clicking at the end of the password field) to paste in the TOTP token.

**Reply by ljlzzzzz** [reply]
OK, thanks
